projects/packages/identity-crisis/src/class-package-version.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * The Package_Version class.
+ *
+ * @package automattic/jetpack-identity-crisis
+ */
+
+namespace Automattic\Jetpack\IdentityCrisis;
+
+/**
+ * The Package_Version class.
+ */
+class Package_Version {
+
+	const PACKAGE_VERSION = '0.18.0-alpha';
```

Comment:
There's already a version number at https://github.com/Automattic/jetpack/blob/3a4ca69e8aff2771fb67f4213636a9915a9ab6f0/projects/packages/identity-crisis/src/class-identity-crisis.php#L29
and that one is set up with the tooling to be managed
https://github.com/Automattic/jetpack/blob/3a4ca69e8aff2771fb67f4213636a9915a9ab6f0/projects/packages/identity-crisis/composer.json#L62-L64

Reply:
Oh thanks for catching that! Is removing package version from class-identity-crisis.php and redirecting composer to class-package -version.php is enough?

Reply:
That' s true. Maybe moving the `send_package_version_to_tracker` method to class-identity-crisis.php and keeping everything there would be less disruptive. 
But, also less clean as it's different from other packages.

Reply:
@anomiex I changed the approach and moved version tracking to class-identity-crisis.php. It feels safer for now. Also, we may be moving the whole IDC package to the connection, so I'm trying to avoid too many breaking changes. 
Does this change look good to you?